debugger/src/main/java/org/codice/acdebugger/breakpoints/ImpliesBreakpointProcessor.java

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.acdebugger.breakpoints;
+
+import com.google.common.collect.ImmutableSet;
+import com.sun.jdi.ObjectReference;
+import com.sun.jdi.StackFrame;
+import com.sun.jdi.ThreadReference;
+import com.sun.jdi.request.EventRequest;
+import java.security.Permission;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.codice.acdebugger.api.BreakpointProcessor;
+import org.codice.acdebugger.api.Debug;
+import org.codice.acdebugger.impl.BreakpointInfo;
+import org.codice.acdebugger.impl.BreakpointLocation;
+
+/**
+ * Old version of a breakpoint processor that was attempting to intercept all calls to {@link
```

Comment:
❓ Why is there an "old version" in this rewrite? Shouldn't we be able to remove it?

Reply:
This was my first take at trying to address in a generic fashion the implies() breakpoint. It does hold valuable information if we ever look into including it.

I kinda wanted to keep it around.